staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -60,11 +81,11 @@ func getRoutes(input io.Reader) ([]Route, error) {
 			continue
 		}
 		fields := strings.Fields(line)
```

Comment:
OK, but it is kind-of obvious from L75-77 and variable names, what these fields are. This file has a header line, but much of the line is irrelevant to our needs. Not sure of how much value this adds.